src/test/java/org/apache/cassandra/sidecar/AbstractHealthServiceTest.java

```diff
@@ -129,9 +130,9 @@ private WebClientOptions getWebClientOptions()
         return options;
     }
 
-    @DisplayName("Should return HTTP 503 Failure when check=False")
+    @DisplayName("Should return HTTP 200 OK when check=False")
```

Comment:
Wrong display name. "200 but NOT_OK"
But.. if the changes to the health endpoint is not necessary for the patch. Please just revert.